bindings/bind_include.hh

```diff
@@ -92,8 +60,8 @@ namespace rascal {
       using Map = std::map<std::string, std::string>;
       Map mapping = {{"StructureManager", ""},
                      {"Adaptor", ""},
-                     {"RepresentationManager", ""},
-                     {"FeatureManager", ""}};
+                     {"Calculator", ""},
+                     {"Calculator", ""}};
```

Comment:
not really part of this pull request, but is there a reason why this is a structure and not just a const mapping? When I see a structure, I think even you want to do some partial specialization or the object has a state, which could change during runtime. I would change the code 

>    
    /**
     * Mapping used to replace all occurences of the first string with the
     * second string in the class titles in the python binding module names. 
     *
     * first: string which should be replaced
     * second: the replaced with string  
     */
    const std::map<std::string, std::string> module_name_replace_with = {
        {"StructureManager", ""},
        {"Adaptor", ""},
        {"Calculator", ""},
        {"Calculator", ""}};

Reply:
map is not an intrinsic type so this is the way I found to make it work.

Reply:
@agoscinski Do you have a suggestion?

Reply:
commited, if everyone is happy with it, resolved